### DISCUSSION TITLE
Which variant to pick? bullseye or other

### DISCUSSION THREAD

**Discussion by mellson** [community/community#6567] [Codespaces] [answered]
I'm not sure I understand the variant set in `devcontainer.json`, the comments say to use -bullseye variants on local arm64/Apple Silicon. I need to run node 16 on a mix of intel and apple silicon machines. Which variant should I pick then for the codespace?

**Comment by Chuxel** [comment, marked as answer]
@mellson Pick `-bullseye` if you need to use a dev container locally using VS Code Remote - Containers on an Apple Silicon mac in addition any intel machines you may have (macOS, Windows, or Linux). This image works on any of them. 

In general, go with `bullseye` (aka Debian 11) unless you hit a compatibility problem and need to use an older OS. (See [here for nitty-gritty](https://github.com/microsoft/vscode-dev-containers/issues/558) details).

If you're only using Codespaces, you really don't need to worry about this as much since the service runs everything runs on Intel hardware behind the scenes today.

That help?

**Reply by mellson** [reply]
Very helpful, thanks a lot!
